lib/health_cards/attribute_filters.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module HealthCards
+  # Handles behavior related to removing disallowed attributes from FHIR Resources
+  module AttributeFilters
+    def self.included(base)
+      base.send :include, InstanceMethods
```

Comment:
I'd get rid of the `InstanceMethods` submodule and just stick those in `AttributeFilters`, since it is already being included.